index.bs

```diff
@@ -11159,11 +11107,14 @@ Here's how the above diagram could be constructed in JavaScript:
 <h2 id="DynamicLifetime">
 Dynamic Lifetime</h2>
 
-<h3 id="dynamic-lifetime-background">
+<h3 id="dynamic-lifetime-background" class="informative">
 Background</h3>
 
-<em>This section is non-normative. Please see <a href="#lifetime-AudioContext">AudioContext lifetime</a> and <a href="#lifetime-AudioNode">AudioNode lifetime</a> for normative
-requirements.</em>
+<em>This section is non-normative.</em>
+
+Note: The normative description of {{AudioContext}} and {{AudioNode}} lifetime characteristics is
```

Comment:
I think this Note should go before the paragraph that says the section is non-normative.